js/app/widgets/eknWebview.js

```diff
@@ -174,11 +182,20 @@ const EknWebview = new Knowledge.Class({
             let scheme = GLib.uri_parse_scheme(uri);
             if (scheme !== null && this.EXTERNALLY_HANDLED_SCHEMES.indexOf(scheme) !== -1) {
                 if (scheme === 'license') {
+                    // Create a license viewer
+                    if (this._license_view === undefined) {
```

Comment:
I would say `if (!this._license_view)`, comparing things to `undefined` can be problematic in JS